composer.json

```diff
@@ -106,6 +106,7 @@
         ],
         "post-autoload-dump": [
             "Illuminate\\Foundation\\ComposerScripts::postAutoloadDump",
+            "@php artisan optimize --clever --dont-confirm=assume-no",
```

Comment:
Will this command be run if updating via GUI? If so, what happens with the prompt then?

Reply:
The prompt is skipped because we use `--dont-confirm=assume-no`.